src/plugins/intel_gpu/src/kernel_selector/core/cache/cache.json

```diff
@@ -20980,6 +21009,35 @@
             "9267895309185863228": ["convolution_gpu_bfyx_os_iyx_osv16", 1]
         },
         "24": {
+            "13641327981711194102" : ["convolution_gpu_bfyx_gemm_like", 2],
```

Comment:
Are those line represent the same conv cases as lines for v2? If yes, then they're not needed here.

Reply:
Okay, I see, I will remove them. they are same conv cases as lines for v2.

Reply:
Remove duplicated v1 lines. thanks.